apps/foundation-app/src/app/product/components/product/product.component.html

```diff
@@ -1,4 +1,24 @@
-<div class="product-container">
-  {{product.id}}
-  {{product.cost}}
-</div>
\ No newline at end of file
+<div class="product">
+  <div class="product__description-container">
+    <div class="product__brand">{{product.brand}}</div>
+    <div class="product__name">{{product.name}}</div>
+    <div class="product__price">${{product.price}}</div>
+  </div>
+  <div class="product__image-gallery-wrapper">
+    <image-gallery></image-gallery>
+  </div>
+  <div class="product__images-wrapper">
```

Comment:
I think the functionality should remain the same between desktop and mobile, this will reduce code complexity long term and keep ux more consistent. @xelaint what do you think?